lms/templates/footer.html

```diff
@@ -58,6 +58,7 @@
               <a href="${link['url']}">${link['title']}</a>
             </li>
           % endfor
+          <li><a href="https://www.edx.org/?utm_medium=affiliate_partner&amp;utm_source=opensource-partner&amp;utm_content=open-edx-partner-footer-link&amp;utm_campaign=open-edx-footer">${_("Take free online courses at edX.org")}</a></li>
```

Comment:
I'm wondering if the URL should be a configuration setting. I don't love having hard coded links, although this one seems unlikely to change. We also don't want operators to change it, so maybe making it configurable would encourage them to do so. I could go either way on this.